kanin/src/lib.rs

```diff
@@ -67,7 +67,7 @@
 //!
 //! #[tokio::main]
 //! async fn main() -> kanin::Result<()> {
-//!     App::new()
+//!     App::new(())
```

Comment:
I don't know if this is the place, but I really think some documentation of how to use state, including the derive, is missing.

Reply:
Added an example here too.